### PR TITLE
Meta-txn endpoint docs

### PR DESCRIPTION
**Pull request by fabioberger**
This PR adds the docs for the meta-txn endpoints.

### CODE REVIEW

File: mdx/api/index.mdx [hunk 1]
```diff
@@ -635,6 +635,113 @@ Response will be an array of token symbols and the respective price in `sellToke
 
 </CodeTabs>
 
+# Meta Transactions
+
+This API makes it easy for integrators to offer their users a meta-transaction trading experience. It provides a superset of the liquidity available via the Swap API (additionally including RFQ liquidity), and also allows integrators to submit the transaction on behalf of their users. This allows them to subsidize the gas cost and protocol fees, removing the need for their users to own Ether.
```
Comment (fragosti) [inline mdx/api/index.mdx]: would link to Swap API.

File: mdx/api/index.mdx [hunk 2]
```diff
@@ -635,6 +635,113 @@ Response will be an array of token symbols and the respective price in `sellToke
...
+| `sellTokenAddress`  | The ERC20 token address of the token you want to sell with quote.                                                                                                                               |                                                                                                          |
+| `zeroExTransactionHash`           | The 0x transaction hash that must be signed by the trader.                   |
+| `zeroExTransaction`           | The 0x transaction struct containing the encoded calldata that calls a 0x exchange contract method with the best-priced orders as a parameter.                   |
+| `sources`           | The percentage distribution of `buyAmount` or `sellAmount` split between each liquidity source. Ex: `[{ name: '0x', proportion: "0.8" }, { name: 'Kyber', proportion: "0.2"}]`                   |
+| `orders`            | [Signed orders](/docs/api#signed-order) that data is comprised of.                                                                                                                               |
+
+## POST /meta_transaction/v0/fill
```
Comment (fragosti) [inline mdx/api/index.mdx]: As mentioned in code review, think it should be `/submit`

File: mdx/api/index.mdx [hunk 3]
```diff
@@ -635,6 +635,113 @@ Response will be an array of token symbols and the respective price in `sellToke
...
+
+### Request
+
+| HTTP Header          | Description                                                                                                                                                                                                         |
+| -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `0x-api-key`          | A 0x api key identifying the caller to the API.                                                                                                      |
+
+
+| Query Param          | Description                                                                                                                                                                                                         |
```
Comment (fragosti) [inline mdx/api/index.mdx]: Not a query param

File: mdx/api/index.mdx [hunk 4]
```diff
@@ -635,6 +635,113 @@ Response will be an array of token symbols and the respective price in `sellToke
...
+| -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `signature`          | (Required) 0x transaction hex-encoded signature.                                                                                                      |
```
Comment (fragosti) [inline mdx/api/index.mdx]: We typically only call out if it is optional.

File: mdx/api/index.mdx [hunk 5]
```diff
@@ -635,6 +635,113 @@ Response will be an array of token symbols and the respective price in `sellToke
...
+#### Case (2)
+
+HTTP Status: 502
+
+| Field               | Description                                                                                                                                                                                      |
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `code`             | 106 |
+| `reason`         | "Unable to submit transaction on behalf of taker" |
+| `ethereumTransaction`         | The raw Ethereum transaction the caller must sign and broadcast to execute the trade encoded in the 0x transaction. |
+
```
Comment (fragosti) [inline mdx/api/index.mdx:776]: No examples?

File: mdx/api/index.mdx [hunk 6]
```diff
@@ -635,6 +635,113 @@ Response will be an array of token symbols and the respective price in `sellToke
...
+| Field               | Description                                                                                                                                                                                      |
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `ethereumTransactionHash`             | The hash of the signed Ethereum transaction executing the 0x transaction. |
+| `signedEthereumTransaction`         | The hex-encoded signed Ethereum transaction that can be re-broadcast if desired (0x API has already broadcast it as well). |
+
+
+#### Case (2)
+
+HTTP Status: 502
```
Comment (fragosti) [inline mdx/api/index.mdx]: Shouldn't it be a 403?

File: mdx/guides/unlocking-meta-transaction-swaps.mdx [hunk 7]
```diff
@@ -0,0 +1,92 @@
+# Unlocking meta-transaction swaps With 0x API
+
+0x protocol has had native support for a trade to occur as a meta-transaction since V2, and we have now added this support to 0x API. This guide will walk you though integrating with 0x API in such a way that enables your users to benefit from a meta-transactions flow.
+
+**Note:** This tutorial assumes your users already have their allowances set to 0x protocol.
```
Comment (fragosti) [inline mdx/guides/unlocking-meta-transaction-swaps.mdx]: Nice. Could link to https://0x.org/docs/guides/how-to-set-your-token-allowances#setting-allowances-on-etherscan

**Review by fragosti — COMMENTED** [review]
🙏

File: mdx/api/index.mdx (hunk 5 above)
Comment (fabioberger) [reply mdx/api/index.mdx:776]: Added!

File: mdx/api/index.mdx (hunk 6 above)
Comment (fabioberger) [reply mdx/api/index.mdx]: Yep, fixed.

**Review by fragosti — APPROVED** [review]

**Pull request merged**